include/shark/Models/Trees/CARTClassifier.h

```diff
@@ -86,6 +86,15 @@ class CARTClassifier : public AbstractModel<RealVector,LabelType>
 			ar & r;
 			ar & g;
 		}
+		NodeInfo() {}
+		explicit NodeInfo(std::size_t nodeId) : nodeId(nodeId) {}
+		NodeInfo(std::size_t nodeId, LabelType const& label) : nodeId(nodeId), label(label) {}
+		NodeInfo(std::size_t nodeId, LabelType && label)
+				: nodeId(nodeId), label(std::move(label)) {}
+		NodeInfo(NodeInfo const&) = default;
+		NodeInfo(NodeInfo &&) = default;
```

Comment:
hi the issue is only that you can not write "= default". You can still implement it by yourself (by using the naive implementation).